modules/data-entry/src/main/frontend/src/questionnaire/Question.jsx

```diff
@@ -129,8 +129,9 @@ function Question (props) {
             ( labels ?
               <List>
                 { labels.map( (item, idx) => {
+                  let value = defaultDisplayFormatter ? defaultDisplayFormatter(item, idx) : item;
```

Comment:
This would also work:

```suggestion
                  let value = defaultDisplayFormatter?.(item, idx) ?? item;
```

Reply:
However if the formatter decides to display `null` we fall back to `item`, which changes the original behavior. Please disregard.